bigquery_schema_generator/generate_schema.py

```diff
@@ -113,7 +114,7 @@ def __init__(
         # If CSV, preserve the original ordering because 'bq load` matches the
         # CSV column with the respective schema entry using the position of the
         # column in the schema.
-        self.sorted_schema = (input_format in {'json', 'dict'})
+        self.sorted_schema = (input_format in {'json', 'dict'}) and not keep_input_sort_order
```

Comment:
If you run `$ make flake8` at the root of the project, you'll see that it will complain about being longer than 80 columns. Break this into smaller lines using something like:

```
self.sorted_schema = (
  (input_format in {'json', 'dict'})
  and not keep_input_sort_order
)